esmf_regrid/schemes.py

```diff
@@ -314,18 +418,26 @@ def __init__(self, mdtol=0):
             data is tolerated while ``mdtol=1`` will mean the resulting element
             will be masked if and only if all the overlapping elements of the
             source grid are masked.
+        src_mask : bool, default=False
+            If True, derive a mask from source cube which will tell :mod:`ESMF`
+            which points to ignore.
+        tgt_mask : bool, default=False
```

Comment:
I've decided to avoid this since at this point a scheme can be initialised without reference to a particular grid. A scheme can be initialised and then used with different source/target grids so I want to avoid tying it to any particular grid at this stage.